modules/tinymce/src/plugins/lists/test/ts/browser/IndentTest.ts

```diff
@@ -522,6 +522,48 @@ describe('browser.tinymce.plugins.lists.IndentTest', () => {
     assert.equal(editor.selection.getNode().nodeName, 'LI');
   });
 
+  it('TINY-10268: a `list` inside a `li` surrounded by 2 not element should allow the user to intend/outdent its `li`', () => {
+    const editor = hook.editor();
+    editor.setContent(`<ul>
+      <li>
+        Dedent me
+        <ul>
+          <li class="to-indent">or try to dedent me 1</li>
+          <li>or try to dedent me 2</li>
+        </ul>
+        <span>abc</span>
+      </li>
+    </ul>`);
+
+    LegacyUnit.setSelection(editor, '.to-indent', 0);
```

Comment:
Avoid adding new tests that use LegacyUnit I want to get rid of that some day. :smile: `TinySelection.setCursor` is the new API.

Reply:
done, sorry :sweat_smile: